subt/trace.py

```diff
@@ -49,11 +53,11 @@ def reverse(self):
         self.trace.reverse()
 
     def add_line_to(self, xyz):
-        last = self.trace[-1]
-        size = distance3D(last, xyz)
-        dx, dy, dz = xyz[0] - last[0], xyz[1] - last[1], xyz[2] - last[2]
-        for i in range(1, int(size/self.step)):
-            s = self.step * i/size
-            self.trace.append((last[0] + s*dx, last[1] + s*dy, last[2] + s*dz))
+        if len(self.trace) > 0:
```

Comment:
Is it legit to call `add_line_to` when there is no previous point in the trace?

Reply:
I think yes ... but it is true, that this is exactly the case where it will fail in `subt/main.py`